.goreleaser.yaml

```diff
@@ -87,21 +47,51 @@ builds:
       - muslc
       - netgo
 
+dockers:
+  - image_templates:
+      - iconcommunity/{{.ProjectName}}:{{ .Tag }}-amd64
+    dockerfile: goreleaser.dockerfile
+    use: buildx
+    build_flag_templates:
+      - "--label=org.opencontainers.image.created={{.Date}}"
+      - "--label=org.opencontainers.image.title={{.ProjectName}}"
+      - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+      - "--label=org.opencontainers.image.version={{.Version}}"
+      - "--platform=linux/amd64"
+  - image_templates:
+      - iconcommunity/{{.ProjectName}}:{{ .Tag }}-arm64v8
+    dockerfile: goreleaser.dockerfile
```

Comment:
Why aren't we using the existing dockerfile and using scratch ? using debian significantly increases the docker image size.

Reply:
Because we're using CGO and that makes it system dependent.

And we've already built the binary just need to copy it.

Reply:
can you give me a breakdown on the docker size ?

Reply:
AMD64: 72.02MB
ARm64: 300.25MB

Reply:
> Why aren't we using the existing dockerfile and using scratch ? using debian significantly increases the docker image size.

more info here:

https://goreleaser.com/errors/docker-build/

I had to struggle for couple of days for this mistake when I was using scratch image.

Reply:
I meant before and after docker image as we already had docker image based on scratch. Based on the increment of the image size, we can discuss further.

Reply:
Never used it, thats not based on scratch but alpine one unlike prod build.

Reply:
That dockerfile can be safely purged without consequences.

Reply:
that is the basis for all e2e tests and cannot be purged but if we are to make docker for relay, the best one would be to utilize similar pattern.